system/src/Grav/Common/Utils.php

```diff
@@ -980,7 +980,7 @@ public static function arrayUnflattenDotNotation($array, $separator = '.')
      */
     public static function pathPrefixedByLangCode($string)
     {
-        if (strlen($string) <= 3) {
+        if (strlen($string) < 3) {
```

Comment:
Language prefix can have more than 2 letters in it.

Reply:
Yes, that didn't change.

Reply:
What change are you suggesting? The original logic was that the URL must have at least 4 characters to be even considered, and I changed it to 3 to match `/en` too instead of only `/en/`... None of this changes anything for prefixes that are longer, since those were not short-circuit-rejected in the first place... (They would lead an URL longer than 3 characters)

Reply:
It occurs to me now that you may have looked at this change out of context. The full code is:

```php
if (strlen($string) < 3) {
  return false;
}
```

Reply:
i actually removed this completely, as languages are not restricted in their length now, can be anything really.